test/test_contrib.py

```diff
@@ -371,8 +371,53 @@ def test_exception(self, device, dtype):
             stitcher(input1, input2)
 
 
-class TestHistMatch:
+class TestConvDistanceTransform:
+    @pytest.mark.parametrize("kernel_size", [3, 5, 7])
+    def test_smoke(self, kernel_size, device, dtype):
+        B, C, H, W = 1, 3, 100, 100
+        input1 = torch.rand(B, C, H, W, device=device, dtype=dtype)
+        distance_transformer = kornia.contrib.ConvDistanceTransform(kernel_size)
+
+        output1 = distance_transformer(input1)
+        output2 = kornia.contrib.conv_distance_transform(input1, kernel_size)
+
+        assert isinstance(output1, torch.Tensor)
+        assert isinstance(output2, torch.Tensor)
+        assert output1.shape == input1.shape
+
```

Comment:
I miss a couple of `test_value` to verify results vs expected. We also use this tests to showcase as a small snippet in the docstrings.